operators/advanced_network/python/check_system.py

```diff
@@ -0,0 +1,632 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import re
+import subprocess
+import sys
+
+
+def setup_logging():
+    """
+    Configures the logging settings.
+    """
+    logging.basicConfig(
+        level=logging.INFO,
+        format="%(asctime)s - %(levelname)s - %(message)s",
+        datefmt="%Y-%m-%d %H:%M:%S",
+    )
+
+
+def parse_args():
+    """
+    Parses command-line arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description="Check system tuning for ANO performance",
+        epilog=(
+            "Examples:\n"
+            "  python check_system.py -cpu-freq       # Check CPU frequency governor\n"
+            "  python check_system.py -mrrs     # Check MRRS settings for NVIDIA NICs\n\n"
+            "  python check_system.py -mps      # Check max payload size settings for NVIDIA NICs\n\n"
```

Comment:
Outdated.

Though it raises a good point to discuss: if we add "setters", the script should be named something else than `check_system` IMO. The setters and name change could wait for another MR to not block this.

Reply:
Maybe tune_system.py? I think the setters should stay together since there's a lot of commonality

Reply:
I agree they should stay together. we can revisit on the next round